Makefile

```diff
@@ -36,7 +37,10 @@ VERSION := $(shell git describe --tags --always --dirty)
 
 SRC_DIRS := cmd pkg # directories which hold app source (not vendored)
 
-ALL_ARCH := amd64 arm arm64 ppc64le
+ALL_ARCH := amd64 386
+ALL_ARCH_CONTAINER := amd64
```

Comment:
I get what ALL_ARCH is, but this I have no idea - would need a comment at the least.